analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -479,7 +498,19 @@ def add_arguments_to_parser(parser):
                                     "OWN RISK!")
 
     logger.add_verbose_arguments(parser)
-    parser.set_defaults(func=main)
+    parser.set_defaults(func=main,
+                        func_process_config_file=process_config_file)
+
+
+def process_config_file(args):
+    """
+    Handler to get config file options.
+    """
+    if 'config_file' not in args or not os.path.exists(args.config_file):
+        return None
+
+    cfg = load_json_or_empty(args.config_file, default={})
```

Comment:
I think it would be nice to print out a message to the user that a config file is used.
It can be easy to forget a hidden file in the home directory.